hooks/post_gen_project.py

```diff
@@ -0,0 +1,28 @@
+import random
+import string
+import os
+
+secret = ''.join(
+    random.choice(string.printable[:94].replace('"', '')) for _ in range(50))
+
+dirs = [
+    f for f in os.listdir('.') if all(
```

Comment:
Ehm.... Ik kan niet eens lezen wat je hier uithaalt :-)
Het lijkt alsof je op creatieve manier kijkt of iets een extensie heeft?

Dat kan iets makkelijker met `os.path.isdir()`. Of met een `os.walk()` er doorheen lopen omdat je toch wel weet dat het ding `settings.py` heet.